nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamServiceImpl.java

```diff
@@ -144,10 +147,10 @@ public void start(ComponentContext context) {
 
     protected void initProcessor(StreamProcessorDescriptor descriptor) {
         if (processors.containsKey(descriptor.getId())) {
-            getLog().error("Processor already initialized: " + descriptor.getId());
+            log.error("Processor already initialized: {}", descriptor.getId());
             return;
         }
-        getLog().info("Init Stream processor: " + descriptor.getId() + " with manager: " + descriptor.config);
+        log.info("Init Stream processor: {} with manager: ", descriptor.getId(), descriptor.config);
```

Comment:
Missing `{}` and could use lambda

Reply:
Usage of lambda will be more verbose as there's no method to get `config` on purpose.